libs/dsb-client-gateway/ui/topics/src/lib/components/Topics/Topics.effects.ts

```diff
@@ -4,10 +4,14 @@ import {
   TopicsModalsActionsEnum,
 } from '../../context';
 import { useTopics } from '@dsb-client-gateway/ui/api-hooks';
+import { TTableComponentAction } from '@dsb-client-gateway/ui/core';
+import { PostTopicDto } from '@dsb-client-gateway/dsb-client-gateway-api-client';
+import { useStyles } from './Topics.styles';
 
 export const useTopicsEffects = () => {
+  const { theme } = useStyles();
   const router = useRouter();
-  const { topics } = useTopics(router.query['namespace'] as string);
+  const { topics, topicsById, isLoading: topicsLoading } = useTopics(router.query['namespace'] as string);
```

Comment:
You can do something like this:
```suggestion
  const { topics, topicsById, isLoading: topicsLoading } = useTopics(router.query[Queries.Namespace] as string);
```